recipes/fast_io/all/conanfile.py

```diff
@@ -0,0 +1,68 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import get, copy
+from conan.tools.layout import basic_layout
+from conan.tools.scm import Version
+import os
+
+required_conan_version = ">=1.52.0"
+
+
+class FastIoConan(ConanFile):
+    name = "fast_io"
+    description = "fast_io is a C++ io exception-safe RAII library based on C++ 20 Concepts. It is at least 10 times faster than cstdio than iostream."
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/cppfastio/fast_io"
+    topics = ("cstdio", "iostream", "io", "header-only")
+    package_type = "header-library"
+    settings = "os", "arch", "compiler", "build_type"
+    no_copy_source = True
+
+    @property
+    def _min_cppstd(self):
+        return 20
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "gcc": "11",
+            "clang": "12",
+            # Apple-Clang 13 is not enough in this case,
+            # partial support for concepts
+            "apple-clang": "14",
+            "Visual Studio": "16",
```

Comment:
```suggestion
            "Visual Studio": "17",
```

Reply:
@RubenRBS 
Thank you for your suggestions.
But these supported versions of these compilers are not released, so these supported versions are not meaningful.

These versions have been fixed since April 21.
https://github.com/cppfastio/fast_io/issues/532

I will try to revert cci to 2023.04.18.